DemoApp/Sources/Components/Gleap/GleapAdapter.swift

```diff
@@ -0,0 +1,74 @@
+//
+// Copyright © 2025 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+import StreamVideo
+#if canImport(Gleap)
+import Gleap
+#endif
+
+final class GleapAdapter {
+
+    var isAvailable: Bool {
+        #if canImport(Gleap)
+        return true
+        #else
+        return false
+        #endif
+    }
+
+    init() {
+        #if canImport(Gleap)
+        Gleap.initialize(withToken: "qsoyideVaZ7ZOehLLl8LqDC6YOPagLvr")
```

Comment:
Is this token safe to be available publicly? What can bad people do with it?

Reply:
Nothing really. They can use to report bugs 😅 fwiw Sentry is the same (we have it publicly available at DemoApp+Sentry.swift)